manifests/poc-data-cluster/prod/spark/iot-packets-silver.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-packets-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      gateway_tmst,
+      oui,
+      net_id,
+      rssi,
+      frequency,
+      snr,
```

Comment:
unit. And make sure it's matching the other silver table for these units

Reply:
Fixed